src/core/Directus/Application/CoreServicesProvider.php

```diff
@@ -553,6 +553,10 @@ protected function getEmitter()
 
                     $type = $field->getType();
                     if (DataTypes::isHashType($type)) {
+                        if (is_array($value) && isset($value['hashed'])) {
+                            $payload->set($key, $value['hashed']);
+                            continue;
+                        }
```

Comment:
This'll break regular `hash` type fields. They way they work right now is to hash the raw value on creation. By not-hashing it when a value already exists, we'll effectively break hash-type support.